src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
```

Comment:
In my opinion, our .container class should be much simpler. Its task should only be to center our content and keep it in the right width.

.container is a class you'll want to use in various places, such as your header or footer section (header may not be in this design case, but in your other projects). Therefore, it is good for him to do his duty (as described above) and do only that, nothing more.

My proposal for a .container class is this:
margin: 0 auto;
max-width: 1024px;
margin - to center content
max-width so that it doesn't go out of bounds
\*  possibly padding so that on mobile devices there is a distance from the edge of the screen, but here we do not have it in the design yet

All other properties like display: flex or also paddings, top-bottom margins should be provided already by a given block such as .header or, as I wrote above .cards